mongodb/README.md

```diff
@@ -0,0 +1,120 @@
+# MongoDB Connector
+
+A Hazelcast Jet connector for MongoDB which enables Hazelcast Jet pipelines to 
+read/write data points from/to MongoDB.
+
+## Getting Started
+
+### Installing
+
+The MongoDB Connector artifacts are published on the Maven repositories. 
+
+Add the following lines to your pom.xml to include it as a dependency to your project:
+
+```
+<dependency>
+    <groupId>com.hazelcast.jet.contrib</groupId>
+    <artifactId>mongodb</artifactId>
+    <version>${version}</version>
+</dependency>
+```
+
+or if you are using Gradle: 
+```
+compile group: 'com.hazelcast.jet.contrib', name: 'mongodb', version: ${version}
+```
+
+### Usage
+
+#### As a Batch Source
+
+MongoDB batch source (`MongoDBSources.mongodb()`)  executes the 
+query and emits the results as they arrive.
+
+Following is an example pipeline which queries from `myCollection` which is in 
+`myDatabase`, filters the documents which has `val` field greater than or equal
+to `10`, applies the projection so that only the `val` and `_id` fields are
+returned and logs them.
```

Comment:
Is the projection and filtering applied server-side or in Jet?

Reply:
both applied on server-side